engine/title.asm

```diff
@@ -3,17 +3,654 @@ INCLUDE "constants.asm"
 SECTION "Title screen", ROMX[$5D8C], BANK[$01]
 
 IntroSequence:: ; 5d8c
-    ; TODO
+	ld hl, GameFreakLogo ;E4000
+	ld a, $39
+	call FarCall_hl
+	jr c, TitleSequenceStart ;5dae
+	ld a, [wOpeningType] ;cc38
+	and a
+	jr z, .opening_sequence_start
+	
+.pikachu_minigame_start
+	ld hl, PikachuMiniGame ;E0000
+	ld a, $38
+	call FarCall_hl
+	jr TitleSequenceStart ;5dae
 
+.opening_sequence_start
+	ld hl, OpeningSequence ;E432F
+	ld a, $39
+	call FarCall_hl
+
+TitleSequenceStart::
+	call TitleSequenceInit
+	ld hl, SetTitleBGDecorationBorder ;91D2
+	ld a, $02
+	call FarCall_hl
+	
+.loop
+	call TitleScreenMain ;5fb8
+	jr nc, .loop
+	
+	call ClearPalettesAndWait ;361e -clear palettes and wait
+	call ClearSprites
+	ld a, $01
+	ldh [$FFDE], a
```

Comment:
If this label was unknown, you would define it as an unknown hram label (`hFFDE`), but it is actually `hBGMapMode`

Reply:
Done.